lib/services/server/utils/deviceRegistrationUtils.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
```

Comment:
License headers needed in all new files. You can copy paste from other existing file and add your self in a "Modified by:" entry at the bottom of the header.

Reply:
Fixed in 376e415

(Same here)